contents/blog/the-posthog-array-1-43-0.md

```diff
@@ -0,0 +1,123 @@
+---
+date: 2023-01-16
+title: "Array 1.43.0: Massive performance improvements!"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+> **Need to update a self-hosted instance?** [Check the runbook docs for instructions](/docs/runbook/upgrading-posthog)!
+
+## PostHog 1.43.0 release notes
+
+**Release highlights:**
+
+- [New: Performance improvements](#new-performance-improvements)
+- [New: Feature flag variant overrides](#new-feature-flag-variant-overrides)
+- [New: Export recordings to file](#new-export-recordings-to-file)
+- [New: Better insight searching](#new-better-insight-searching)
+- [New: Show event/property counts in persons modal](#new-show-eventproperty-counts-in-persons-modal)
```

Comment:
This doesn't point to a paragraph yet – just to be explicit, should I write one or would you prefer to take over this?

Reply:
Nope, I've written one and got a gif. Something is up with this PR which is giving me grief though. Let me try to add it in again.